# GPIO pin number
FAULT: 26
PWM2: 19
PWM1: 13
DIR2: 6
DIR1: 5
EN: 22